src/docs/structural_variant_allele.md

```diff
@@ -0,0 +1,21 @@
+# Structural Variant Allele
+Currently models non-breakend SVs
+
+
+
+| Field             | Type            | Description
+|-------------------|-----------------|---------------------
+| name               | string         | Name of the allele - from an accessioning repository(dbVar, DGVa)
+| alternative_names  | array of External Reference or [] | Alternative names + source info
+| type               | string                     | This is always StructuralVariantAllele
+| allele_type        | OntologyTermMetadata| SO type eg. insertion, deletion
+| slice              | Slice    | Slice describing the coordinates of the allele
+| copy_number        | integer | Add the copy number for the CNV supporting structural variants when available
```

Comment:
The description may be clearer as -
'For CNVs, the number of copies of the repeat unit present in this allele, otherwise null'

We should also state the type is integer or null.